### PR TITLE
Add remote dir support

### PR DESCRIPTION
**Pull request by eoehen**
- Support new docker volume `remote`.
- Add environment variable `PUSH_REMOTE_MODE`.
- Implemented two modes `move` and `copy`.

### CODE REVIEW

File: backup.sh [hunk 1]
```diff
@@ -70,21 +73,36 @@ do
...
+    find $WORKDIR -type f -name "*.$CURRENT_DB.trn"     -exec cp {} $TARGETDIR \;
+  fi
+
+  # Push to remote directory
+  if [ "$PUSH_REMOTE_MODE" = "move" ] || [ "$PUSH_REMOTE_MODE" = "copy" ]; then
+    echo "Push backup to remote directory"
+    find $WORKDIR -type f -name "*.$CURRENT_DB.bak"     -exec cp {} $REMOTEDIR \;
+    find $WORKDIR -type f -name "*.$CURRENT_DB.trn"     -exec cp {} $REMOTEDIR \;
+    find $WORKDIR -type f -name "*.$CURRENT_DB.zip"     -exec cp {} $REMOTEDIR \;
+    find $WORKDIR -type f -name "*.$CURRENT_DB.tar.gz"  -exec cp {} $REMOTEDIR \;
```
Comment (pfeigl) [inline backup.sh:95]: Couldn't we just also use the same find as is beeing used in the cleanup:

`find $TARGETDIR -type f -name "*.$CURRENT_DB.*" -exec rm {} \;`

I think atleast we should be somewhat consistant and either just take all based on the name schema or define all file extensions everywhere.

**Review by pfeigl — CHANGES_REQUESTED** [review]
One thing I'm missing from this implementation is the fact, that the remote is never cleaned up.

We could add an option for this. But tbh I think if cleanup is active and push_remote is active, we should just blindly also assume, that we should cleaup the remote.

What do you think?

**Review by georgesgoetz — APPROVED** [review]

File: backup.sh (hunk 1 above)
Comment (eoehen) [reply backup.sh:95]: @pfeigl Thank you very much for your review. I am agree with both.

**Comment by eoehen** [comment]
@pfeigl I have processed your feedback. Do you agree?

**Comment by eoehen** [comment]
Extracted from PR #25 by @pfeigl

File: backup.sh [hunk 2]
```diff
@@ -70,38 +73,47 @@ do
...
+      echo "Cleanup target directory"
+      find $TARGETDIR -type f -name "*.$CURRENT_DB.*" -exec rm {} \;
+    fi
   fi
 
-  # Delete intermediate directory
-  echo "Delete intermediate directory"
+  # Cleanup intermediate directory
+  echo "Cleanup intermediate directory"
+  find $WORKDIR -type f -name "*.$CURRENT_DB.*" -exec rm {} \;
```
Comment (pfeigl) [inline backup.sh:102]: Couldn't we just `rm -rf` instead of deleting the files and than using `rmdir`

**Review by pfeigl — APPROVED** [review]
Small feedback on deleting the intermediate folder. Other than that, LGTM.

File: backup.sh (hunk 2 above)
Comment (eoehen) [reply backup.sh:102]: @pfeigl done.

**Comment by eoehen** [comment]
@pfeigl Thank you very much for review.

**Pull request merged**